inst/shiny/modules/tab_visuals.R

```diff
@@ -250,8 +278,20 @@ tab_visuals_server <- function(id, data, grouping_vars, res_nca) {
         choices = sort(studyid_choices)
       )
 
+      # Update pcspec mean choices
+      pcspec_choices <- data() %>%
+        pull(PCSPEC) %>%
+        unique()
+
+      updateSelectInput(
+        session,
+        "pcspec_mean",
+        choices = sort(pcspec_choices),
+        selected = pcspec_choices[1]
```

Comment:
nitpick(non-blocking) / question: The same as before, but here we are sorting the choices, but picking the selected from an unsorted vector. Is that intended? It might happen that the selected option will not be the first option of the sorted choices.
```suggestion
        choices = sort(pcspec_choices)
```